projects/web-wallet/rpc-protocol.md

```diff
@@ -0,0 +1,330 @@
+# 1. Introduction
+
+This document outlines the remote procedure call (RPC) methods for a decentralized application (dApp) to interact with Hathor wallets.
+
+The design is currently a minimum viable product (MVP), addressing only known needs from our community. It should be noted that this API design may change as the platform evolves and additional needs arise.
+
+The idea here is to partially replicate the headless wallet API as it is already being used by many partners and has been validated to work on their use cases
+
+# 2. Reference-level explanation
+
+## 2.1 RPC Methods
+
+### 2.1.1 `htr_createTx`
+
+The `htr_createTx` method creates a new transaction based on parameters. It will select the `utxos` automatically, mine and send the transaction
```

Comment:
Why is this `createTx` and not `sendTx`? Isn't it used to send txs?

Reply:
It's not only sending transactions, but it's also building it according to the parameters

I could rename it to `createAndSendTx`, what do you think?

Reply:
@obiyankenobi I agree, refactored in 7621950fa0e0af8e089271f13919f9af3f5cf3de